doc/reference/filters.md

```diff
@@ -168,37 +131,33 @@ When there are multiple servers in a pool, the Proxy can do a load balance betwe
-| mainPool       | [proxy.PoolSpec](#proxyPoolSpec)               | Main pool of backend servers                                                                                                                                                                                                                                                                                        | Yes      |
-| candidatePools | [][proxy.PoolSpec](#proxyPoolSpec)             | One or more pool configuration similar with `mainPool` but with `filter` options configured. When `Proxy` get a request, it first goes through the pools in `candidatePools`, and if one of the pools filter in the request, servers of this pool handles the request, otherwise, the request is pass to `mainPool` | No       |
-| mirrorPool     | [proxy.PoolSpec](#proxyPoolSpec)               | Definition a mirror pool, requests are sent to this pool simultaneously when they are sent to candidate pools or main pool                                                                                                                                                                                          | No       |
-| failureCodes   | []int                                          | HTTP status codes need to be handled as failure                                                                                                                                                                                                                                                                     | No       |
-| compression    | [proxy.CompressionSpec](#proxyCompressionSpec) | Response compression options                                                                                                                                                                                                                                                                                        | No       |
-| mtls           | [proxy.MTLS](#proxymtls)            | mTLS configuration | No |
-| maxIdleConns    | int                                           | Controls the maximum number of idle (keep-alive) connections across all hosts. Default is 10240 | No |
-| maxIdleConnsPerHost    | int                                    | Controls the maximum idle (keep-alive) connections to keep per-host. Default is 1024               | No |
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| pools | [proxy.ServerPoolSpec](#proxyserverpoolspec) | The pool without `filter` is considered as `mainPool`, other pools with `filter` are considered as `candidatePools`. When `Proxy` get a request, it first goes through the pools in `candidatePools`, and if one of the pools' filter matches the request, servers of this pool handle the request, otherwise, the request is passed to `mainPool` | Yes |  
```

Comment:
```suggestion
| pools | [proxy.ServerPoolSpec](#proxyserverpoolspec) | The pool without `filter` is considered the main pool, other pools with `filter` are considered candidate pools, and a `Proxy` must contain exactly one main pool. When `Proxy` gets a request, it first goes through the candidate pools, and if one of the pool's filter matches the request, servers of this pool handle the request, otherwise, the request is passed to the main pool. | Yes |  
```